requirements.txt

```diff
@@ -1,15 +1,11 @@
 numpy==1.23.2
```

Comment:
clean up all these imports

Only add imports that YOU PERSONALLY are importing in your files

Reply:
imports are cleaned